action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
   github_token:
     description: "Usually <dollar sign>{{ secrets.GITHUB_TOKEN }}"
     required: true
+  opened_only:
```

Comment:
should we rename it to `isLimitedToOpenedReopened` .
make it type boolean
default false
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
I'm struggling with a better name, because it's too late to switch to camelCase.  We have 48 projects using this with_underscore_separation.  `limit_to_pr_opened`?